misk/src/main/kotlin/misk/web/actions/UpstreamResourceInterceptor.kt

```diff
@@ -76,31 +90,39 @@ class UpstreamResourceInterceptor(
   /**
    * Maps URLs requested against this server to URLs of servers to delegate to
    *
-   * localPathPrefix: `/_admin/`
-   * upstreamBaseUrl: `http://localhost:3000/`
+   * url_path_prefix: `/_admin/`
+   * server_url: `http://localhost:3000/`
    *
    * An incoming request then for `/_admin/config.js` would route to `http://localhost:3000/_admin/config.js`.
    */
+  // Make register tab a subclass of mapping
```

Comment:
what’s this comment mean?

Reply:
Was taking notes of what I thought you meant but turns out I misunderstood.